topics/netlify/index.md

```diff
@@ -0,0 +1,11 @@
+---
+display_name: Netlify
+github_url: https://github.com/netlify
+logo: netlify.png
+short_description: Netlify: Develop & deploy the best web experiences
+topic: netlify
+url: https://netlify.com
+---
+
+Nelify is a powerful serverless platform with an intuitive git-based workflow. 
```

Comment:
```suggestion
Netlify is a powerful serverless platform with an intuitive git-based workflow. 
```

Reply:
@BlakeWilliams Kindly Check #2215. I have made the changes